doc/coding_standard.md

```diff
@@ -120,6 +121,29 @@ Shared pointers (`std::shared`) are preferred over raw pointers.
 Exception to this rule is when the code interacts with a 3rd party library like PETSc where
 shared pointers simply don't exist.
 
+### Conditionals
+
+A space should be used after the keyword in a conditional statement. There
+is no space inside parentheses. The statement block should not be enclosed
+in braces if the condition fits on a single line.
+
+```c++
+if (a == b)
+  a = 0;
+```
+
+If the condition spans multiple lines, the statement block may be enclosed in
+braces for clarity.
+
+```c++
+ if (std::none_of(my_container.begin(),
+                  my_container.end(),
+                  [](int val) { return val == 0; }))
+  {
+    return;
+  }
```

Comment:
The `{` and `}` should be indented on `i`, not `f`

Reply:
That whole block was a cut-and-paste disaster. Fixed.